NEWS

```diff
@@ -43,6 +43,17 @@ to these changes. This was done in a backwards-compatible way, however
 there is one special case: Users using VNC with X.509 support enabled,
 will need to be running at least QEMU 2.5. See #1342 for details.
 
+GHC 7.10 and 8.x compatibility
+++++++++++++++++++++++++++++++
+
+Ganeti 2.16.0 could only be built using GHC versions prior to 7.8, as
+GHC 7.10 and later versions introduced breaking API changes that made
+the build fail.
+
+This version introduces support for newer GHC versions and Ganeti is now
+known to build with GHC 8.0, 8.2 and 8.4. Additionally, compatibility
```

Comment:
"known to build with GHC 8.0, 8.2 and 8.4" - this is at first read unclear if compat with 7.x too. I'd explicitly mention "and 8.4, on top of the current 7.8".

Reply:
I rephrased the whole section to also cover the libraries, thanks! @iustin: would you mind setting the CLA flag so that we can merge this?

Reply:
Thanks, new version looks good, resolving.